tests/test_advanced_options.py

```diff
@@ -77,3 +78,24 @@ def test_bulk_analysis(analysis_data):
         assert len(current_report['depItems']) >= 0, "No dependencies were found"
         violations = [item for item in current_report['rulesets'] if item.get('violations')]
         assert len(violations) > 0, "No issues were found";
+
+# Validation for Jira ticket MTA-3779
+def test_private_repo(analysis_data):
```

Comment:
`test_analysis_of_private_repo`

Reply:
tackle-test-app-public is public repo.

Reply:
The idea here is that we are speaking about some personal repo, doesn't matter if it is available or not for everybody. BTW, without proper github credentials it will not be abalized properly.

Reply:
The repo is public, but some of the dependencies require the user to provide correct credentials to be downloaded